### ISSUE TITLE
Extension Error: Failed to construct 'URL': Invalid URL

### ISSUE THREAD

**Issue by dgracehmh** [GoogleChrome/lighthouse#1210]
**Lighthouse Version**: 1.3.0
**Chrome Version**: 55.0.2883.95
**Error Message**: Failed to construct 'URL': Invalid URL
**Stack Trace**:
 ```TypeError: Failed to construct 'URL': Invalid URL
    at artifacts.AnchorsWithNoRelNoopener.usages.filter.anchor (chrome-extension://blipmdconlkpinefehnmjammfjpmpbjk/scripts/lighthouse-background.js:689:25)
    at Array.filter (native)
    at Function.audit (chrome-extension://blipmdconlkpinefehnmjammfjpmpbjk/scripts/lighthouse-background.js:689:8)
    at Promise.resolve.then._ (chrome-extension://blipmdconlkpinefehnmjammfjpmpbjk/scripts/lighthouse-background.js:11365:20)```

**Comment by wardpeet** [comment]
@dgracehmh  could you provide us with an url please.
do you have any chrome plugins installed? Do you have any options deselected on the extension?

// dupe of #1194

**Comment by dgracehmh** [comment]
@wardpeet i'm running lighthouse against an internal url, structure below
http://xxx.xxxxx.xxx.xxxxx.com/xxxx/login/?state=http%3A%2F%2Fxxx.xxxxxx.xxx.xxxxxx.com%2Fxxxx%2F%23%2F

I can run lighthouse fine against other urls so I guess the problem is not with other extensions. In any case I also have ZenHub, Poper Blocker, Attacat Cookie audit tool.

The tool looks great btw!
Cheers

**Comment by wardpeet** [comment]
Thanks for the feedback, error should be fixed in #1218
